types/vergeSense/cloudConnector/space_availability.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://akenza.io/vergeSense/cloudConnector/space_availability.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "space_availability",
+  "title": "Space availability",
```

Comment:
what is the difference to the occupancy topic?

Reply:
The status is sent from the sensor directly, while occupancy/occupied is generated within the script based on movement.